.travis.yml

```diff
@@ -89,32 +92,33 @@ install:
   # I will remove the sphinx downgrade. Change setup.py requirements when it
   # be the case, Katta
   - echo "------- Install Dependencies -------"
-  - pip3 install -U pip setuptools
-  - pip3 install Jinja2 PyYAML pytest sympy --user
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-      pip3 install -U --user sphinx==2.4.4 nbsphinx>=0.3.2 m2r sphinx-rtd-theme jupyter;
-    fi
```

Comment:
`jupyter` is not in the list and hence the question : are we running notebooks now? If not, we should make sure to execute those without error (as they will become live notebooks soon).

Reply:
I do not remember why we changed this. @ohm314 ?

Reply:
jupyter will be installed with `setup.py` it is a build dependency, so it belongs there. You see that the `nbsphinx` package is used in sphinx, which should take care of this. I think also it can be seen here if I'm not mistaken: https://travis-ci.org/github/BlueBrain/nmodl/jobs/691131098#L3159
I think the config: https://nbsphinx.readthedocs.io/en/0.7.0/usage.html#nbsphinx_execute
should be correct: https://github.com/BlueBrain/nmodl/blob/ad458721f938090357512ab87dfea5145a1a7a67/docs/conf.py#L74

Reply:
as by private conversations:

> now sphinx is in charge of generating the docs and running the notebooks